.github/workflows/runtime-snapshot-build.yml

```diff
@@ -11,6 +11,8 @@ on:
       - "runtime-snapshot-*"
       - "runtime-taurus-*"
       - "runtime-mainnet-*"
+  pull_request:
```

Comment:
prefer not building the docker images. 
Instead we can run them as part of the of the rust.yml CI